3DSView/src/main/java/eu/livotov/labs/android/d3s/D3SView.java

```diff
@@ -157,6 +160,16 @@ public void onPageStarted(WebView view, String url, Bitmap icon)
                 }
             }
 
+            @Override
+            public void onPageFinished(WebView view, String url) {
+                super.onPageFinished(view, url);
+
+                if (url.toLowerCase().contains(postbackUrl.toLowerCase())) {
+                    return;
+                }
+                view.loadUrl(String.format("javascript:window.%s.processHTML(document.getElementsByTagName('html')[0].innerHTML);", JavaScriptNS));
```

Comment:
Commenting out this line fixes the responsiveness issue.

Reply:
I assume that re-introduces the original issue on Chrome 71?

Reply:
Yeah it totally breaks the library. I'm trying to avoid this on first page load to see if that improves things.

Reply:
Winner, it does. I'll push those changes now.

Reply:
aaanndd ... I can't push to your branch

Reply:
```diff
diff --git a/3DSView/src/main/java/eu/livotov/labs/android/d3s/D3SView.java b/3DSView/src/main/java/eu/livotov/labs/android/d3s/D3SView.java
index 94af1ab..3620bd0 100755
--- a/3DSView/src/main/java/eu/livotov/labs/android/d3s/D3SView.java
+++ b/3DSView/src/main/java/eu/livotov/labs/android/d3s/D3SView.java
@@ -72,6 +72,8 @@ public class D3SView extends WebView
      * When set to <b>true</b>, SSL certificate errors and self-signed certificates errors will be ignored.
      */
     private boolean debugMode = false;
+    
+    private boolean initialPageLoadCompleted = false;
 
     /**
      * Url that will be used by ACS server for posting result data on authorization completion. We will be monitoring
@@ -167,9 +169,14 @@ public class D3SView extends WebView
                 if (url.toLowerCase().contains(postbackUrl.toLowerCase())) {
                     return;
                 }
-                view.loadUrl(String.format("javascript:window.%s.processHTML(document.getElementsByTagName('html')[0].innerHTML);", JavaScriptNS));
+                
+                if(!initialPageLoadCompleted) {
+                               initialPageLoadCompleted = true;
+                } else {
+                               view.loadUrl(String.format("javascript:window.%s.processHTML(document.getElementsByTagName('html')[0].innerHTML);", JavaScriptNS));
+                }
             }
-;
+
             public void onReceivedError(WebView view, int errorCode, String description, String failingUrl)
             {
                 if (!failingUrl.startsWith(postbackUrl))
@@ -332,6 +339,7 @@ public class D3SView extends WebView
     {
         urlReturned = false;
         postbackHandled.set(false);
+        initialPageLoadCompleted = false;
 
         if (authorizationListener != null)
         {
```

Reply:
Seems to work on both Chrome 70 and 71